android/src/main/java/io/sentry/RNSentryModule.java

```diff
@@ -108,6 +116,27 @@ public void startWithOptions(final ReadableMap rnOptions, Promise promise) {
                 } catch (Exception e) {
                     // We do nothing
                 }
+
+                // Add on the correct event.origin tag.
+                // it needs to be here so we can determine where it originated from.
+                SdkVersion sdkVersion = event.getSdk();
```

Comment:
We could merge this on its own PR already, separately from the scope sync.

Reply:
I think this is merged, just didn't do a rebase on the latest master yet.

Reply:
Sorry about all these issues I'm not used to making PRs

Reply:
Thanks for all the comments; I will get everything resolved tomorrow!